src/providers/image-loader.ts

```diff
@@ -414,12 +414,19 @@ export class ImageLoader {
    * @returns {Promise<string>} Returns a promise that resolves with the local path if exists, or rejects if doesn't exist
    */
   private getCachedImagePath(url: string): Promise<string> {
+    // Check if we're running with livereload
+    let _isDev: boolean = (window['IonicDevServer'] != undefined);
     return new Promise<string>((resolve, reject) => {
 
       // make sure cache is ready
       if (!this.isCacheReady) {
         return reject();
       }
+      
+      // if we're running with livereload, ignore cache and call the resource from it's URL
+      if(!!_isDev){
```

Comment:
`!!` is not required.. since `window['IonicDevServer'] != undefined` or `typeof window['IonicDevServer'] !== 'undefined'` will always return a `boolean`